src/Runner.Worker/ExecutionContext.cs

```diff
@@ -559,14 +597,10 @@ public void Progress(int percentage, string currentOperation = null)
         }
 
         // This is not thread safe, the caller need to take lock before calling issue()
-        public void AddIssue(Issue issue, string logMessage = null)
+        public void AddIssue(Issue issue, ExecutionContextLogOptions logOptions)
```

Comment:
Can we make `logOptions` defaults to `ExecutionContextLogOptions.Default` so you don't need to change all existing callers?
`public void AddIssue(Issue issue, ExecutionContextLogOptions logOptions = ExecutionContextLogOptions.Default)`

Reply:
ignore this, for now, check the other comment at the top. 😄